packages/calypso-e2e/src/rest-api-client.ts

```diff
@@ -16,21 +16,43 @@ interface RequestParams {
 	body?: BodyInit;
 }
 interface Site {
-	ID: string;
+	ID: number;
 	name: string;
 	description: string;
 	URL: string;
-	site_owner: string;
+	site_owner: number;
 }
-interface MySitesResponse {
+interface AllSitesResponse {
 	sites: Site[];
 }
+interface CalypsoPreferencesResponse {
+	calypso_preferences: {
+		recentSites: number[];
+	};
+}
+interface MyAccountInformationResponse {
+	ID: number;
+	username: string;
+	email: string;
+}
+export interface NewUserResponse {
+	code: number;
+	body: {
+		success: boolean;
+		user_id: number;
+		username: string;
+		bearer_token: string;
+	};
+}
```

Comment:
At some point I may have to move these into its own Types file.

Reply:
Totally fair! I think they're fine here for now FWIW 😄